torchvision/datasets/folder.py

```diff
@@ -202,8 +202,11 @@ def make_dataset(
     ) -> List[Tuple[str, int]]:
         return make_dataset(directory, class_to_idx, extensions=extensions, is_valid_file=is_valid_file)
 
-    @staticmethod
-    def _find_classes(dir: str) -> Tuple[List[str], Dict[str, int]]:
+    def find_classes(self, dir: str) -> Tuple[List[str], Dict[str, int]]:
+        """
+        Same as :func:find_classes. This method can be overridden to only consider 
```

Comment:
backticks + docstring format :)

```suggestion
        """Same as :func:`find_classes`.
        
        This method can be overridden to only consider 
```